src/cli/analyze/analyze-cli-command.ts

```diff
@@ -37,6 +37,33 @@ Please follow and contribute to the discussion at:
 		);
 	}
 
+	if (config.format === "webtypes") {
+		// Allow object being passed as JSON from command line
+		if (typeof config.webtypesConfig === "string") {
+			config.webtypesConfig = JSON.parse(config.webtypesConfig);
```

Comment:
maybe be worth catching errors here and raising a cli error in case of invalid JSON

Reply:
Fixed